### ISSUE TITLE
Understanding 1.3.5 - relative link in the SC boxout at the top isn't working

### ISSUE THREAD

**Issue by patrickhlauke** [w3c/wcag#442]
On https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html in the grey box-out at the top (which I presume is automagically pulled from WCAG 2.1 core document) the link `<a href="#input-purposes">Input Purposes for User Interface Components section</a>` isn't working (as it's relative link in the main WCAG 2.1 spec document, which should be adapted to point to `https://www.w3.org/TR/WCAG21/#input-purposes` instead)

**Comment by michael-n-cooper** [comment]
Fixed via https://github.com/w3c/wcag/commit/a924475e9bbd586394c79d91bdeca1e55e9caf79.
